### PR TITLE
feat: Update for new server side config

### PR DESCRIPTION
**Pull request by fleck**
## Give us some context

* What problems does this PR solve?
We're updating the docs for the new server side config functionality.

Preview link: https://deploy-preview-19497--docs-website-netlify.netlify.app/docs/apm/agents/manage-apm-agents/configuration/server-side-agent-configuration/

### CODE REVIEW

**Review by mchavez-newrelic — APPROVED** [review]
Looks good! @sujitnewrelic we'd like to hold off on merging this PR until the new server side config is ready. We'll reach out once this PR is officially ready to merge.

**Review by sujitnewrelic — APPROVED** [review]
lgtm

**Pull request merged**
